migtests/tests/pg/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -317,6 +317,10 @@
 					"SqlStatement": "ALTER TABLE ONLY schema2.sales_region\n    ADD CONSTRAINT sales_region_pkey PRIMARY KEY (id, region);"
 				}
 			]
+		},
+		{
+			"FeatureName": "Foreign tables",
+			"Objects": []
```

Comment:
Why no objects?

Reply:
I haven't added a test in assessment test for that as it needs a table from another database and so on. So thought anyways @shubham-yb is adding sample schemas which has this so should be validated there